packages/components/popover/src/PopoverTrigger/PopoverTrigger.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { usePopoverContext } from '../PopoverContext';
+
+/**
+ * PopoverTrigger opens the popover. It must be an interactive element.
+ */
+export const PopoverTrigger: React.FC = (props) => {
```

Comment:
I think we need to update the code style because it was kind agreed that we would prefer not to use `React.FC` and type only the props the component will use
even `children`